JSDemos/Demos/DateBox/Formatting/description.md

```diff
@@ -1 +1,17 @@
-This demo shows how to&nbsp;use a&nbsp;locale-dependent format (or&nbsp;default format), <a href="/Documentation/ApiReference/Common/Object_Structures/format/#type">built-in format</a>, <a href="/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String">LDML&nbsp;pattern</a>, and a&nbsp;format with literal characters to&nbsp;specify the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat">displayFormat</a>. Note that when the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior">useMaskBehavior</a> option is&nbsp;set to&nbsp;<b>true</b>, the <b>DateBox</b> formats the display value and also controls user input.
\ No newline at end of file
+To customize the **DateBox** format, use the [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat) property. This demo illustrates the following use cases of this property:
+
+- **Locale-dependent format**     
+The DateBox formats its value according to the user's locale. This is the default behavior. Refer to the following help topic for more information: [Localize Dates, Numbers, and Currencies](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies)
+
+- **Built-in predefined format**    
+DevExtreme supports [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type). Set the **displayFormat** property to the respective format.    
```

Comment:
```suggestion
DevExtreme supports a number of [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type). Those are simple strings or shortcuts that define commonly used date formats so that you don't need to spell out a complex expression. Review the linked document for the list of values you can assign to **displayFormat**.  This demo uses the "shortdate" format. 
```